app/views/admin/users/show.html.erb

```diff
@@ -1,50 +1,112 @@
-<div class="grid gap-6">
-  <div class="crayons-card p-6">
-    <h2 class="d-inline">
+<%# this is for pawel's testing purpose to emulate tabs... %>
+<% current_tab = "overview" # overview|emails|notes|reports|flags %>
+<% if true %>
+  <%= render "header" %>
+  <%= render "navigation" %>
+
+  <% if current_tab == 'overview' %>
+    <div class="crayons-card p-3 s:p-4 m:p-7">
```

Comment:
I intentionally do not include some of these wrappers/containers (in this entire file) inside the partial because they define the general layout - so if we ever need to change we would do it in one files instead of gazillion partials.